gradle/libs.versions.toml

```diff
@@ -18,6 +18,8 @@ micrometer-version = "1.15.2"
 binary-compatibility-validator-version = "0.18.1"
 kotlin-multiplatform-bignum-version = "0.3.10"
 kotlinx-datetime-version = "0.6.1"
+publish-plugin-version = "2.0.0"
```

Comment:
naming: the plugin is called `plugin-publish`, so should this be `plugin-publish-plugin-version`?